.appveyor.yml

```diff
@@ -22,8 +20,8 @@ install:
 services:
   - postgresql111
 before_build:
-  - '"C:\Program Files\PostgreSQL\11\bin\psql" -c "CREATE EXTENSION citext"'
-  - '"C:\Program Files\PostgreSQL\11\bin\psql" -c "CREATE EXTENSION postgis"'
+  - '"C:\Program Files\PostgreSQL\11\bin\psql" -d postgresql://postgres:Password12!@localhost -c "CREATE EXTENSION citext"'
```

Comment:
Yes, it breaks some connection tests. Since a variable takes precedence over a provide password callback it's used and test fails. Therefore, I moved out connection settings from variables.

Reply:
It doesn't seem correct for the variable to take precedence over the callback, since the callback is explicitly set up in code whereas the variable is just an environmental thing...

Reply:
Fixed.